m1-init: update project for the first homework

DimazikSkyter opened the pull request:
1) наличие корневого проекта - в master;
2) наличие подпроектов на Kotlin - m1;
3) правильно настроенный .gitignor и отсутствие мусорных файлов в проекте - m1; 4) версии библиотек в gradle.properties - done in m1.

DimazikSkyter committed:
m1-init: create files for first projects and fix incorrect includes.

DimazikSkyter committed:
m1-init: fix all points
1) наличие корневого проекта - в master;
2) наличие подпроектов на Kotlin - m1;
3) правильно настроенный .gitignor и отсутствие мусорных файлов в проекте - m1;
4) версии библиотек в gradle.properties - done in m1.

DimazikSkyter committed:
Delete gradlew

fix master

DimazikSkyter committed:
Delete gradlew.bat

DimazikSkyter committed:
Delete gradlew

svok commented on build.gradle.kts:
```diff
@@ -17,12 +17,20 @@ dependencies {
 
```
Версия плагина для котлина тоже должна уехать в gradle.properties

svok reviewed (commented):
Дмитрий, хорошая работа. Рекомендую доделать вынос версий в gradle.properties

DimazikSkyter committed:
m1-init: fix all points
1) наличие корневого проекта - в master;
2) наличие подпроектов на Kotlin - m1;
3) правильно настроенный .gitignor и отсутствие мусорных файлов в проекте - m1;
4) версии библиотек в gradle.properties - done in m1.

DimazikSkyter committed:
Merge branch 'master' into m1-init
